package.json

```diff
@@ -38,7 +38,7 @@
         "eslint-plugin-testing-library": "^4.2.0",
         "husky": "^7.0.0",
         "jest": "^26.4.2",
-        "lerna": "^3.22.0",
+        "lerna": "^4.0.0",
```

Comment:
why do we need to update this?
i think it should be deeply check since it is entangled with our project infrastructure.

Reply:
Lerna version 4 has not made changes that might make our project to behave differently. (Changelog of version 4: https://github.com/lerna/lerna/blob/main/CHANGELOG.md)

npm was treating the older version as a critical vulnerabality and it had to updated.